charts/sentry-apps/templates/application.yaml

```
@@ -0,0 +1,66 @@
+{{ template "argoconfig.application" (list . "sentry-apps") }}
+
+{{- define "sentry-apps" }}
+metadata:
+  {{- .Values.spec.metadata | toYaml | nindent 2 }}
+spec:
+  destination:
+    {{- with .Values.spec.destination.namespace }}
+    namespace: {{ . }}
+    {{- end }}
+    {{- with .Values.spec.destination.server }}
+    server: {{ . }}
+    {{- end }}
+  project: {{ .Values.spec.project }}
+  source:
+    {{- if .Values.spec.source.path }}
+    path: {{ .Values.spec.source.path }}
+    {{- else }}
+    chart: {{ .Values.spec.source.chart }}
+    {{- end }}
+    repoURL: {{ .Values.spec.source.repoURL }}
+    targetRevision: {{ .Values.spec.source.targetRevision | quote }}
```

`helm create` usually creates a chart that uses `.Chart.AppVersion` so the version does not need to be duplicated:
```suggestion
    targetRevision: {{ .Values.spec.source.targetRevision | default .Chart.AppVersion | quote }}
```

---

Much nicer indeed.